scss/atoms/typography/_typography.scss

```diff
@@ -12,11 +12,11 @@
 body {
 	font: $normal #{ map-get( $defaultFont, font-size ) }/1.5 $sans;
 
-	@media ( max-width: 800px ) {
+	@media ( min-width: $viewport-xs ) {
```

Comment:
Okay we currently do not explicitly define any viewports, we just use these breakpoints to apply styles to them cascadingly in mobile first manner, so the default font size in line 13 would apply to `???`
so to us its basically 
* less than breakpoint-xs (lets call them breakpoints now to avoid confusion)(This would imply 0-399) the font size in line 13 would apply to this
* more than or equal to breakpoint-sm 
* more than ...  breakpoint-md
* anything more ... than breakpoint lg
Right now this is the way we are distributing our viewport sizes